backend/pyproject.toml

```diff
@@ -26,7 +26,7 @@ dependencies = [
   "alembic==1.13.2",
   "appdirs",
   "cachetools",
-  "fastapi>=0.101.0,<0.112.4",
+  "fastapi[all]>=0.101.0,<0.112.4",
```

Comment:
I added the `[all]` because I was getting some weird mypy errors because `email-validator` and `orjson` were missing. However, I just changed it back, i.e. removed the `all`, here and it seems to work fine again. So I think this should be fixed, but I also agree that adding all of this makes no sense, so if needed in the future we should try to add as few as possible.